### ISSUE TITLE
Unnecessary force calculation at energy_loss.backward()

### ISSUE THREAD

**Issue by IgnacioJPickering** [aiqm/torchani#669]
There may be some unnecessary loss calculations when calling loss.backwards() since the gradient may be propageted to the coordinates, even when not needed. Maybe not an issue anymore however, but pinning this here since it needs investigating.
